src/log.h

```diff
@@ -223,5 +245,28 @@ namespace librealsense
             }
         }
     };
+#else //BUILD_EASYLOGGINGPP
+    struct log_message
+    {
+        std::string error_msg;
+        log_message(): error_msg("") {}
```

Comment:
If you want to explicitly initialize, error_msg() is enough -- the default ctor gives an empty string. Giving it the "" means the ctor will try to call strlen(), etc -- extra work that's not needed.